autogpt_platform/backend/backend/server/routers/v1.py

```diff
@@ -43,14 +52,10 @@ def execution_scheduler_client() -> ExecutionScheduler:
 v1_router = APIRouter(prefix="/api")
 
 
-v1_router.dependencies.append(Depends(auth_middleware))
-
-
```

Comment:
why dropping this?

Reply:
There were up to 3 levels of redundant `Depends(auth_middleware)`: this line, then on the sub-routers, and on the individual endpoints. Unfortunately there is no way to exempt an individual endpoint from these rules, so I had to remove all of the entries affecting the webhook ingress endpoint.